### PR TITLE
vmui: add flag for default timezone setting

### PR DESCRIPTION
**Pull request by Loori-R**
Implemented a new feature for timezone management in vmui. This includes:

1. **Introduction of Configuration Flag**:
   - Implementation of the `--vmui.defaultTimezone` flag to set a default timezone.
2. **Enhancement of User Interface Timezone Selection**:
   - Addition of the UTC timezone option in the vmui interface.
   - Inclusion of the timezone specified by the `--vmui.defaultTimezone` flag in the vmui timezone selection dropdown.
3. **Fallback to Browser's Local Timezone**:
   - If the `--vmui.defaultTimezone` flag is unset or invalid, vmui defaults to the user's browser local timezone.
4. **User Preference Priority**:
   - Clarification that user-selected timezones in the vmui interface override the default timezone set by the `--vmui.defaultTimezone` flag.

ref: #5375

image

### CODE REVIEW

File: app/vmselect/vmui.go [hunk 1]
```diff
@@ -65,6 +67,13 @@ func handleVMUICustomDashboards(w http.ResponseWriter) error {
 	return nil
 }
 
+func handleVMUITimezone(w http.ResponseWriter) error {
+	timezone := *vmuiDefaultTimezone
```
Comment (dmitryk-dk) [inline app/vmselect/vmui.go:71]: I think we can validate timezone before return it to the client
```golang
timeLoc, err := time.LoadLocation("ABC-3")
if err != nil {
  fmt.Println(err)
}
```

**Review by dmitryk-dk — COMMENTED** [review]
Overall looks good! Have some comment

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

File: app/vmselect/vmui.go [hunk 2]
```diff
@@ -65,6 +68,16 @@ func handleVMUICustomDashboards(w http.ResponseWriter) error {
 	return nil
 }
 
+func handleVMUITimezone(w http.ResponseWriter) error {
+	tz, err := time.LoadLocation(*vmuiDefaultTimezone)
+	if err != nil {
+		return fmt.Errorf("cannot load timezone %q: %w", *vmuiDefaultTimezone, err)
+	}
+	response := fmt.Sprintf(`{"timezone": "%s"}`, tz)
```
Comment (valyala) [inline app/vmselect/vmui.go:76]:
```suggestion
	response := fmt.Sprintf(`{"timezone": %q}`, tz)
```

**Review by valyala — APPROVED** [review]

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@Loori-R , thanks for the pull request!

**Pull request merged**